tests/qos/files/qos.yml

```diff
@@ -807,15 +807,15 @@ qos_params:
                     dscp: 3
                     ecn: 1
                     pg: 3
-                    pkts_num_trig_pfc: 13012
-                    pkts_num_trig_ingr_drp: 13252
+                    pkts_num_trig_pfc: 12955
```

Comment:
Why are there adjustments in numbers? Do we have changes in the buffer settings?

Reply:
Yes. the numbers are little different in latest SAI. Changes in params for single ToR are already checked in. This PR is to handle dual ToR.